packages/bff/src/utilities/errorMappers.ts

```diff
@@ -17,4 +17,19 @@ export const reversePurposeUpdateErrorMapper = (
     .with("purposeNotFound", () => HTTP_STATUS_NOT_FOUND)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
 
+export const getSelfcareErrorMapper = (error: ApiError<ErrorCodes>): number =>
+  match(error.code)
+    .with("selfcareEntityNotFilled", () => HTTP_STATUS_INTERNAL_SERVER_ERROR)
+    .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const getSelfcareUserErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with("selfcareEntityNotFilled", () => HTTP_STATUS_INTERNAL_SERVER_ERROR)
+    .with("userNotFound", () => HTTP_STATUS_NOT_FOUND)
+    .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const attributeEmptyErrorMapper = (): number =>
+  HTTP_STATUS_INTERNAL_SERVER_ERROR;
```

Comment:
This could also just be `emptyErrorMapper`, it's not specific to "attribute"

```suggestion
export const emptyErrorMapper = (): number =>
  HTTP_STATUS_INTERNAL_SERVER_ERROR;
```

Reply:
Absolutely correct, thanks!